### DISCUSSION TITLE
I already have a portfolio website what should i do now?

### DISCUSSION THREAD

**Discussion by lohitkolluri** [microsoft/SATechnicalOnboarding#57] [Questions]
I already have a portfolio website what should i do now?

**Comment by Akintunde76** [comment]
Please show me how you got your portfolio website up and running. @lohitkolluri 

You must complete the Onboarding survey and share your portfolio website url there.

**Reply by lohitkolluri** [reply]
> Please show me how you got your portfolio website up and running. @lohitkolluri 
> 
> 
> 
> You must complete the Onboarding survey and share your portfolio website url there.

I used azure static web app service and made an azure workflow to build a production copy and deploy it automatically whenever something is pushed. If any other doubt you can reach me at lohitkolluri@gmail.com
